src/Moryx.Model.Sqlite/SqliteDatabaseConfig.cs

```diff
@@ -47,7 +47,7 @@ public override string Database
         }
 
         /// <inheritdoc />
-        [DataMember, Required, DefaultValue("Data Source=.\\db\\<DatabaseName>.db")]
+        [DataMember, Required, DefaultValue("Data Source=.\\db\\<DatabaseName>.db;Pooling=false")]
```

Comment:
Instead of making this option public/a general setting, you could try to use it only for deletion